projects/canopy/src/lib/accordion/docs/accordion.stories.ts

```diff
@@ -2,16 +2,23 @@ import { CommonModule } from '@angular/common';
 import { Meta, moduleMetadata, Story } from '@storybook/angular';
 
 import { LgButtonModule } from '../../button';
-import { LgAccordionModule } from '../accordion.module';
 import { LgAccordionComponent } from '../accordion.component';
+import { LgAccordionItemComponent } from '../accordion-item/accordion-item.component';
+import { LgAccordionPanelHeadingComponent } from '../accordion-panel-heading/accordion-panel-heading.component';
 
 // This default export determines where your story goes in the story list
 export default {
   title: 'Components/Accordion/Examples',
   component: LgAccordionComponent,
   decorators: [
     moduleMetadata({
-      imports: [ CommonModule, LgAccordionModule, LgButtonModule ],
+      imports: [
+        CommonModule,
+        LgAccordionComponent,
```

Comment:
Again, here you should only import `LgAccordionComponent`. Same for all the others.